iogt/settings/base.py

```diff
@@ -428,3 +426,13 @@
     'wagtailcore.locale': ['language_code'],
     'iogt_users.user': ['username'],
 }
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
+    )
+}
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(days=365),
```

Comment:
How did we decide what value to use here?

Reply:
we can't set the token lifetime to infinity and don't have a mechanism to refresh the auth token from rapidpro so set it to 1 year and after 1 year we will have to update token on rapidpro server 
but we can override simplejwt package to bypass the expiration check for users belonging to `rapidpro_chatbot` group